src/examples/learn_quant/meaning.py

```diff
@@ -0,0 +1,33 @@
+import pandas as pd
+from learn_quant.quantifier import QuantifierModel, QuantifierUniverse
+from itertools import product, combinations_with_replacement, permutations
+from ultk.language.sampling import powerset
+import random
+import argparse
+
+def create_universe(M_SIZE, X_SIZE):
+
+    possible_quantifiers = []
```

Comment:
I'd use `possible_models` instead of `possible_quantifiers`, because the `Referent`s in this `Universe` are models, not quantifiers, which are sets of models